src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java

```diff
@@ -271,6 +286,29 @@ public Result getResult() {
             return result;
         }
 
+        public List<String> getLog(int number) {
+            try {
+                return jobBuild.getLog(number);
+            } catch (IOException ex) {
+                

```

Comment:
Bad practice. You should comment why you suppress the exception and probably write it to system logs. If the method should be public, add Javadoc as well